int64 value
---
bool success
